packages/design-system/src/components/Skeleton/variations/SkeletonInput.tsx

```diff
@@ -0,0 +1,20 @@
+import { StackVertical } from '../../Stack';
+import SkeletonPrimitive, { SkeletonPrimitiveProps } from '../Primitive/Skeleton.Primitive';
+import styles from './SkeletonInput.module.scss';
+import SkeletonParagraph from './SkeletonParagraph';
+import React, { forwardRef, Ref } from 'react';
+
+export type SkeletonInputProps = Omit<SkeletonPrimitiveProps, 'className'>;
+
+const SkeletonInput = forwardRef((props: SkeletonInputProps, ref: Ref<HTMLSpanElement>) => {
+	return (
+		<StackVertical gap="XXS" {...props} ref={ref}>
```

Comment:
The ref and the `Stack` are not aligned... Interesting that this compiles.

Reply:
fixed, but: 
`export const StackVertical = forwardRef((props: StackVerticalProps, ref: React.Ref<any>) => {`